### PR TITLE
Component/photo page

### PR DESCRIPTION
**Pull request by ronnyleibham**
- create interface to capture live camere feed
- create page to build "take a photo" page
- ensure that picture will be created when hitting center button

in next user story base64 formatted image will be reformatted to .jpg and uploaded to DeepVA engine

### CODE REVIEW

File: src/app/components/CamDeviceInterface/CamDeviceInterface.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import CamDeviceInterface from './CamDeviceInterface';
+
+export default {
+  title: 'Component/CamDeviceInterface',
+  component: CamDeviceInterface,
+  parameters: { layout: 'fullscreen' },
+};
+
+export const CamInterface = (): JSX.Element => (
+  <CamDeviceInterface></CamDeviceInterface>
```
Comment (phlgr) [inline src/app/components/CamDeviceInterface/CamDeviceInterface.stories.tsx]:
```suggestion
  <CamDeviceInterface/>
```

File: src/app/components/NavBarCamera/NavBarCamera.stories.tsx [hunk 2]
```diff
@@ -6,4 +6,10 @@ export default {
   parameters: { layout: 'fullscreen' },
 };
 
-export const NavBar2 = (): JSX.Element => <NavBarCamera></NavBarCamera>;
+export const NavBar2 = (): JSX.Element => (
+  <NavBarCamera
+    onClick={function (): void {
+      throw new Error('Function not implemented.');
+    }}
+  ></NavBarCamera>
```
Comment (phlgr) [inline src/app/components/NavBarCamera/NavBarCamera.stories.tsx]:
```suggestion
  />
```

File: src/app/components/NavBarCamera/NavBarCamera.stories.tsx [hunk 3]
```diff
@@ -6,4 +6,10 @@ export default {
   parameters: { layout: 'fullscreen' },
 };
 
-export const NavBar2 = (): JSX.Element => <NavBarCamera></NavBarCamera>;
+export const NavBar2 = (): JSX.Element => (
```
Comment (phlgr) [inline src/app/components/NavBarCamera/NavBarCamera.stories.tsx]:
```suggestion
export const NavBar = (): JSX.Element => (
```
🥴

File: src/app/pages/TakePhoto/TakePhoto.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components';
+import CamDeviceInterface from '../../components/CamDeviceInterface/CamDeviceInterface';
+import NavBarCamera from '../../components/NavBarCamera/NavBarCamera';
+
+export default function TakePhoto(): JSX.Element {
+  function handleTakePhoto() {
+    const width = 1080;
+    const height = 720;
+
+    const video = document.getElementsByTagName('video')[0];
```
Comment (phlgr) [inline src/app/pages/TakePhoto/TakePhoto.tsx:10]: This would be great as a useRef, but it's fine for now!

**Review by phlgr — CHANGES_REQUESTED** [review]
Great job!
Say cheese 📸

File: src/app/components/CamDeviceInterface/CamDeviceInterface.stories.tsx (hunk 1 above)
Comment (ronnyleibham) [reply src/app/components/CamDeviceInterface/CamDeviceInterface.stories.tsx]: Done

File: src/app/components/NavBarCamera/NavBarCamera.stories.tsx (hunk 2 above)
Comment (ronnyleibham) [reply src/app/components/NavBarCamera/NavBarCamera.stories.tsx]: Done

File: src/app/components/NavBarCamera/NavBarCamera.stories.tsx (hunk 3 above)
Comment (ronnyleibham) [reply src/app/components/NavBarCamera/NavBarCamera.stories.tsx]: Done

File: src/app/pages/TakePhoto/TakePhoto.tsx (hunk 4 above)
Comment (ronnyleibham) [reply src/app/pages/TakePhoto/TakePhoto.tsx:10]: I'll keep that on my toDo list ;-)

**Pull request merged**